### DISCUSSION TITLE
Test-it function doesn't populate any data and display an empty UI

### DISCUSSION THREAD

**Discussion by NgoKimPhu** [GitbookIO/community#760] [4. Open API]
Related issue: https://github.com/GitbookIO/gitbook/issues/2525
Example: https://docs.snyk.io/snyk-api/reference/apps
![image](https://github.com/user-attachments/assets/4a2757bb-5d21-4632-aa78-8f6ec1833229)

**Comment by rodrcastro** [comment]
Hey @NgoKimPhu - our team is aware of this and we are working with Scalar to understand this further. I'll keep you posted as I have more to share here :)

**Reply by marclave** [reply]
we are on it! optimistic on a fix today :)

**Comment by MantisClone** [comment]
Was this ever resolved?

I'm facing a similar issue, today. The "Test it" function loads most fields but still lacks the Body Parameters. Precision: The Body Parameters render successfully in the Gitbook OpenAPI block. 
Example: https://docs.request.network/request-network-api/create-and-pay-requests
OpenAPI definition: https://api.request.network/open-api/openapi.json
![image](https://github.com/user-attachments/assets/6ca3e2d4-1661-492b-bb0b-f96ef65e6c95)
![image](https://github.com/user-attachments/assets/269c012f-42aa-4cfb-b94d-49ab20307d02)

**Reply by MantisClone** [reply]
Checking in again. Has this been resolved? If not, will there be a fix anytime soon? @addisonschultz

**Reply by MantisClone** [reply]
We resolved our missing body parameters by adding examples to our OpenAPI definition. :white_check_mark: The body parameters are now populated in the "Test it" dialog.

I still think this is a bug - the body parameters should appear in the "Test it" dialog even if examples are not provided by the OpenAPI definition.
![image](https://github.com/user-attachments/assets/759d8d84-c3ac-43e0-8909-8a74573301e1)
